vyxal/elements.py

```diff
@@ -3081,7 +3115,12 @@ def strict_greater_than(lhs, rhs, ctx):
         (NUMBER_TYPE, str): lambda: int(str(lhs) > rhs),
         (str, NUMBER_TYPE): lambda: int(lhs > str(rhs)),
         (str, str): lambda: int(lhs > rhs),
-    }.get(ts, lambda: int(list(lhs) > list(rhs)))()
+    }.get(
+        ts,
+        lambda: int(
+            bool(list(iterable(lhs, ctx=ctx)) > list(iterable(rhs, ctx=ctx)))
```

Comment:
Can we just use a normal function with if-else branches here? The `lambda :`'s are okay in other elements but this is getting out of hand.

Reply:
It only looks complicated because of the type changing - an if-else would be overkill for this.